services/121-service/src/payments/fsp-integration/commercial-bank-ethiopia/commercial-bank-ethiopia.service.ts

```diff
@@ -314,12 +314,31 @@ export class CommercialBankEthiopiaService
     }
   }
 
-  public async getCommercialBankEthiopiaCredentialsOrThrow(
-    programFinancialServiceProviderConfigurationId: number,
-  ): Promise<RequiredUsernamePasswordInterface> {
+  public async getCommercialBankEthiopiaCredentialsOrThrow({
```

Comment:
"Use programFspConfigId to get config instead of programId"

Looks like the code change is actually the other way around. I am a bit confused...